main_test.go

```diff
@@ -131,12 +129,13 @@ func testSendHeartbeats(t *testing.T, entity, project string) {
 }
 
 func TestTodayGoal(t *testing.T) {
-	apiUrl, router, close := setupTestServer()
+	apiURL, router, close := setupTestServer()
 	defer close()
 
 	var numCalls int
 
-	router.HandleFunc("/users/current/goals/11111111-1111-4111-8111-111111111111", func(w http.ResponseWriter, req *http.Request) {
+	url := "/users/current/goals/11111111-1111-4111-8111-111111111111"
```

Comment:
I don't see that variable being used in other places. Do we need to have the url declared as variable?

Reply:
It's to prevent the line too long linter error